test/jdk/java/util/zip/ZipFile/CorruptedZipFiles.java

```diff
@@ -25,148 +25,170 @@
+     */
+    @BeforeMethod
+    public void setUp() {
+        bad = Arrays.copyOf(good, good.length);
+    }
+
+    @Test
+    public void corruptedENDSIZ() {
         bad[endpos+ENDSIZ]=(byte)0xff;
         checkZipException(bad, ".*bad central directory size.*");
+    }
 
-        err.println("corrupted ENDOFF");
-        bad = good.clone();
+    @Test
+    public void corruptedENDOFF() {
         bad[endpos+ENDOFF]=(byte)0xff;
         checkZipException(bad, ".*bad central directory offset.*");
+    }
 
-        err.println("corrupted CENSIG");
-        bad = good.clone();
+    @Test
+    public void corruptedCENSIG() {
         bad[cenpos]++;
         checkZipException(bad, ".*bad signature.*");
+    }
 
-        err.println("corrupted CENFLG");
-        bad = good.clone();
+    @Test
+    public void corruptedCENFLG() {
         bad[cenpos+CENFLG] |= 1;
         checkZipException(bad, ".*encrypted entry.*");
+    }
 
-        err.println("corrupted CENNAM 1");
-        bad = good.clone();
+    @Test
+    public void corruptedCENNAM1() {
         bad[cenpos+CENNAM]++;
         checkZipException(bad, ".*bad header size.*");
+    }
 
-        err.println("corrupted CENNAM 2");
-        bad = good.clone();
+    @Test
+    public void corruptedCENNAM2() {
         bad[cenpos+CENNAM]--;
         checkZipException(bad, ".*bad header size.*");
-
-        err.println("corrupted CENNAM 3");
-        bad = good.clone();
+    }
+    @Test
+    public void corruptedCENNAM3() {
         bad[cenpos+CENNAM]   = (byte)0xfd;
         bad[cenpos+CENNAM+1] = (byte)0xfd;
         checkZipException(bad, ".*bad header size.*");
-
-        err.println("corrupted CENEXT 1");
-        bad = good.clone();
+    }
+    @Test
+    public void corruptedCENEXT1() {
         bad[cenpos+CENEXT]++;
         checkZipException(bad, ".*bad header size.*");
-
-        err.println("corrupted CENEXT 2");
-        bad = good.clone();
+    }
+    @Test
+    public void corruptedCENEXT2() {
         bad[cenpos+CENEXT]   = (byte)0xfd;
         bad[cenpos+CENEXT+1] = (byte)0xfd;
         checkZipException(bad, ".*bad header size.*");
+    }
 
-        err.println("corrupted CENCOM");
-        bad = good.clone();
+    @Test
+    public void corruptedCENCOM() {
         bad[cenpos+CENCOM]++;
         checkZipException(bad, ".*bad header size.*");
-
-        err.println("corrupted CENHOW");
-        bad = good.clone();
+    }
+    @Test
+    public void corruptedCENHOW() {
         bad[cenpos+CENHOW] = 2;
         checkZipException(bad, ".*bad compression method.*");
+    }
 
-        err.println("corrupted LOCSIG");
-        bad = good.clone();
+    @Test
+    public void corruptedLOCSIG() {
         bad[locpos]++;
         checkZipExceptionInGetInputStream(bad, ".*bad signature.*");
-
-        out.printf("passed = %d, failed = %d%n", passed, failed);
-        if (failed > 0) throw new Exception("Some tests failed");
     }
-
     static int uniquifier = 432;
 
```

Comment:
I am not sure how many people would know what a `uniquifier` is though it is an interesting phrase which I know from SQL Server.  Perhaps consider renaming this variable

Reply:
I'm not sure we need unique file names for this test. Opted for a single ZIP Path instead and removed this variable.